css/main.css

```diff
@@ -1,5 +1,6 @@
 * {
-	font-size: 12px;
+	font-size: 14px;
```

Comment:
Okay, but changes like this - particularly anything that affects the styling of the entire document - should be isolated in a separate PR, so they can be reviewed independently.